recipes/hippomocks/all/test_package/CMakeLists.txt

```diff
@@ -1,10 +1,8 @@
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.15)
 project(test_package CXX)
 
-include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup(TARGETS)
-
-find_package(hippomocks REQUIRED)
+find_package(hippomocks REQUIRED CONFIG)
 
 add_executable(${PROJECT_NAME} main.cpp)
 target_link_libraries(${PROJECT_NAME} PRIVATE hippomocks::hippomocks)
+target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_11)
```

Comment:
Please check this comment: https://github.com/conan-io/conan-center-index/pull/18124#discussion_r1246361883